src/amphtml-email/20_Samples/Advanced_Server_Request.html

```diff
@@ -0,0 +1,94 @@
+<!---
+
+preview: amp4email
+skipValidation: 'true'
+draft: 'true'
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to display data from a server in an email and refresh it subsequently
+after the user makes a selection.
+
+It uses a combination of [`amp-list`](https://www.ampproject.org/docs/reference/components/amp-list) and [`amp-form`](https://www.ampproject.org/docs/reference/components/amp-form)
+that share the same [`amp-mustache`](https://www.ampproject.org/docs/reference/components/amp-mustache) template and uses
+[`amp-bind`](https://www.ampproject.org/docs/reference/components/amp-bind) to connect them together.
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <title>AMP for Email - Advanced Server Request</title>
+</head>
+<!-- ## Body -->
+<!-- -->
+<body>
+  <!--
+    Initialize the global state with no animal selected.
+  -->
+  <amp-state id="animal">
+    <script type="application/json">
+      {
+        "selected": ""
+      }
+    </script>
+  </amp-state>
+  <!--
+    In this example, we use an HTML `select` element to allow the user to make a choice.
+    When the user makes a choice, the `change` event is triggered.
+    This updates `amp-state` and submits the form below.
+  -->
+  <div>
+    <p>Select an animal to update the server response.</p>
+    <select on="change:AMP.setState({ animal: { selected: event.value } }), animal-info.submit">
```

Comment:
Why not put the select into the form to avoid having to bind to an extra variable?

Reply:
Ah got it, it's because you're using the variable to show hide the list. Another option would be to use actions:

`on="change:myForm.Submit,myList.hide`